src/sentry/issues/ingest.py

```diff
@@ -189,3 +192,29 @@ def save_issue_from_occurrence(
         group_info = GroupInfo(group=group, is_new=is_new, is_regression=is_regression)
 
     return group_info
+
+
+def send_issue_occurrence_to_eventstream(
+    event: Event, occurrence: IssueOccurrence, group_info: GroupInfo
+) -> None:
+    event_group = event.for_group(group_info.group)
+    event_group.occurrence = occurrence
+
+    eventstream.insert(
+        event=event_group,
+        is_new=group_info.is_new,
+        is_regression=group_info.is_regression,
+        is_new_group_environment=group_info.is_new_group_environment,
+        primary_hash=event_group.get_primary_hash(),
+        # Not totally sure this is right. Should it be detection time instead?
```

Comment:
Is this comment about the `received_timestamp`? What do we do for performance issues?

Reply:
We use `received` from the event and then if unavailable the event datetime, so I'm just going to do the same here